addons/hr/static/src/core/common/hr_employee_public_model.js

```diff
@@ -0,0 +1,23 @@
+import { Record, fields } from "@mail/core/common/record";
```

Comment:
In several files: @mail/core/common/record is no longer valid, should be from model/export